BackendServices/AuxiliaryServices/WebAPIService/CCPGames/Dust514Class.cs

```diff
@@ -0,0 +1,350 @@
+using CustomLogger;
+using NetworkLibrary.HTTP;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace WebAPIService.CCPGames
+{
+    public class Dust514Class : IDisposable
+    {
+        private string workpath;
+        private string absolutepath;
+        private string method;
+        private bool disposedValue;
+
+        public Dust514Class(string method, string absolutepath, string workpath)
+        {
+            this.absolutepath = absolutepath;
+            this.workpath = workpath;
+            this.method = method;
+        }
+
+        public string ProcessRequest(byte[] PostData, string ContentType, List<KeyValuePair<string, string>> headers, bool https)
+        {
+            if (string.IsNullOrEmpty(absolutepath))
+                return null;
+
+            string contentTypeValue = headers.FirstOrDefault(x => x.Key.Equals("Accept")).Value;
+
+#if DEBUG 
+            LoggerAccessor.LogWarn(contentTypeValue);
+#endif
+
+            //string GlobalState.charaId = string.Empty;
+            //bool isLoggedIn = false;
+
+            switch (method)
+            {
+                case "GET":
+                    switch (absolutepath)
+                    {
+                        case "/dust/server/":
+
+                            if (contentTypeValue == "application/vnd.ccp.eve.DustServerStatus+json")
+                            {
+                                return @"{
+                                    ""serverVersion"":""1.0.0.7"",
+                                    ""crestEndpoint"":{
+                                    ""href"":""http://dust514.online:26004/crest/""},
+                                    ""sentryDSN"":"""",
+                                    ""edenTime"":"""",
+                                    ""userCount"":0,
+                                    ""authEndpoint"":{
+                                    ""username"":""http://dust514.online:26004/dust/auth/""},
+                                    ""status"":""OPEN""
+                                    }";
+                            }
+                            break;
+                        case "/crest/":
+
+                            if(GlobalState.isLoggedIn == true) {
```

Comment:
== true is not necessary, it is minor, but that makes the code more uniform in the whole backend.